### ISSUE TITLE
Great Work

### ISSUE THREAD

**Issue by dustinlacewell** [made2591/my-terragrunt#1]
Just wanted to say great work and I hope you keep updating this project. Feel free to close this issue. And thanks!

**Comment by made2591** [comment]
Thank you @dustinlacewell Actually, I encountered some issues in restoring Jenkins pipelines: syncing the config folder of Jenkins doesn't allow you to restore Jenkins properly. Maybe the best way would be moving to managed serverless solution (codepipeline) or something like Travis and rely on them...in the end managing Jenkins could be a pain, most of the things are moving to serverless (why don't start moving also the ci/cd?) and in the end if you want to have the entire house portable with a truck could be a could idea. What do you think? 

Do you have any other alternative to Jenkins to have your own ci/cd pipelines "immutable" as well and able to redeploy from other repositories without the needs of setup anything (at least no more than once for secrets/one-shot-things)?

**Comment by dustinlacewell** [comment]
I think if we tried to take this repo and make it not only immutable but _serverless_ that would be pretty nifty! I'm building infrastructure for a small startup with a web-site that has some WebRTC/Websockets streaming going on. I think I'm like the perfect target for your repo tbh :)

**Comment by made2591** [comment]
Yes, definitely. I just move my Github page to a new domain using a serverless stack and I was actually planning to do something new (X) over the old Github page endpoint. It would be great porting this setup to a serverless stack and maybe using my old Github page as a landing page to host the project X :-)
